python/Ganga/GPIDev/Credentials/GridProxy.py

```diff
@@ -206,12 +204,10 @@ def chooseCommandSet(self):
         Return value: None
         """
         from Ganga.GPIDev.Lib.GangaList.GangaList import GangaList
-        if "ICommandSet" == self.command._name or isType(self.command, GangaList):
-            if self.voms:
-                self.command = self.vomsCommand
-            else:
-                self.command = self.gridCommand
-        return None
+        if self.voms:
```

Comment:
@egede Currently we only have 'new' tests for `Ganga` not any other modules. I wasn't planning on necessarily running each plugin's tests in a separate process but it would be possible. The problem that this PR is attempting to fix is that some things in Ganga read the config on initial startup (i.e. config variables in `defaults_SomeClass`-type sections) and never read them again, even if they are edited by the user at runtime. If we want to change these on a test-by-test basis then they need to be able to re-read the config if there's no other mechanism to override them.

The problem in the (soon-to-be incoming) Dirac test isn't that it's being polluted by values set in previous tests but rather that without editing the config file and restarting ganga, there's no way to change what command `gridProxy.create()` calls.

Indeed I want the Dirac tests to be in the GangaDirac folder etc.